tools/live-test-cleanup/cleanup.ps1

```diff
@@ -29,12 +29,15 @@ param (
     $Environment = "AzureCloud",
 
     [Parameter()]
-    [switch] $Force
+    [switch] $Force,
+
+    [Parameter(ValueFromRemainingArguments = $true)]
+    $IgnoreUnusedArgguments
```

Comment:
Arguments is misspelled

Reply:
LOL... good thing we are ignoring it :) I'll fix it.